### ISSUE TITLE
Physical volume not present

### ISSUE THREAD

**Issue by ddurieux** [fusioninventory/fusioninventory-for-glpi#504]
---

Author Name: **DuyLong LE** (DuyLong LE)
Original Redmine Issue: 1002, http://forge.fusioninventory.org/issues/1002
Original Date: 2011-07-06

---

Physical volume informations is present in /tmp/*.ocs file, but not present in glpi (Component tab).

```
    <PHYSICAL_VOLUMES>
      <ATTR>a-</ATTR>
      <DEVICE>/dev/vda2</DEVICE>
      <FORMAT>lvm2</FORMAT>
      <FREE>14713</FREE>
      <PE_SIZE>4</PE_SIZE>
      <PV_NAME>sys</PV_NAME>
      <PV_PE_COUNT>7560</PV_PE_COUNT>
      <PV_UUID>4I1GWe-I6za-Dc0i-8eiy-rtN5-CVx4-PGxYCA</PV_UUID>
      <SIZE>31708</SIZE>
    </PHYSICAL_VOLUMES>

```

**Comment by ddurieux** [comment]
---

Original Redmine Comment
Author Name: **DuyLong LE** (DuyLong LE)
Original Date: 2011-07-06T08:13:02Z

---

Some informations :

```
    <LOGICAL_VOLUMES>
      <ATTR>-wi-ao</ATTR>
      <LV_NAME>root</LV_NAME>
      <LV_UUID>WWiUoG-hXJ6-Molf-DTuh-qmYp-ns9U-4dPt1O</LV_UUID>
      <SEG_COUNT>1</SEG_COUNT>
      <SIZE>5997</SIZE>
    </LOGICAL_VOLUMES>
    <LOGICAL_VOLUMES>
      <ATTR>-wi-ao</ATTR>
      <LV_NAME>swap</LV_NAME>
      <LV_UUID>XLHls3-9YeD-Qld4-NKk9-z15w-SImq-VqGvg7</LV_UUID>
      <SEG_COUNT>1</SEG_COUNT>
      <SIZE>2998</SIZE>
    </LOGICAL_VOLUMES>
    <LOGICAL_VOLUMES>
      <ATTR>-wi-ao</ATTR>
      <LV_NAME>var</LV_NAME>
      <LV_UUID>dQdFMx-y9fh-ANma-GjqV-SE0Z-Mnhz-zsxZrx</LV_UUID>
      <SEG_COUNT>1</SEG_COUNT>
      <SIZE>7998</SIZE>
    </LOGICAL_VOLUMES>

```

```
    <VOLUME_GROUPS>
      <ATTR>wz--n-</ATTR>
      <FREE>14713</FREE>
      <LV_COUNT>3</LV_COUNT>
      <PV_COUNT>1</PV_COUNT>
      <SIZE>31708</SIZE>
      <VG_EXTENT_SIZE>4.19</VG_EXTENT_SIZE>
      <VG_NAME>sys</VG_NAME>
      <VG_UUID>ft2uyO-BcmP-T3EL-9GRn-E1QS-SyNk-VMJqX1</VG_UUID>
    </VOLUME_GROUPS>

```

**Comment by ddurieux** [comment]
---

Original Redmine Comment
Author Name: **Gonéri Le Bouder** (Gonéri Le Bouder)
Original Date: 2011-07-21T15:46:12Z

---

fixed in commit:8b2926d1af24ee903af101136ed824075ca87477
